### PR TITLE
Mobly Release 1.12.

### PR DESCRIPTION
**Pull request by xpconanfan**
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/846)

### CODE REVIEW

**Review by mhaoli — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly/846)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @eric100lin and @xpconanfan)

___
*[`CHANGELOG.md` line 26 at r1](https://reviewable.io/reviews/google/mobly/846#-N8REEFw4ySOcbCTX1sl:-N8REEFx9BTzYpH5t4Iu:bmj5kjt) ([raw file](https://github.com/google/mobly/blob/e20286b6f912138a85dc8b0a3f20724b8b12d628/CHANGELOG.md#L26)):*
> ```Markdown
> * Android snippet client v2 based on the new base client
> * Change Mobly's logger level to `DEBUG` via cli arg
> * Termination signal type is now included in result records.
> ```

Delete this period, or use periods for all the four list items?



_Code quote:_
```Markdown
.
```
___
*[`CHANGELOG.md` line 30 at r1](https://reviewable.io/reviews/google/mobly/846#-N8RERY1E0FR-qR8L30c:-N8RERY1E0FR-qR8L30d:b-2guby3) ([raw file](https://github.com/google/mobly/blob/e20286b6f912138a85dc8b0a3f20724b8b12d628/CHANGELOG.md#L30)):*
> ```Markdown
> ### Breaking Changes
> * The old Android snippet client is deprecated
> * The `generate_setup_tests` stage is now `pre_run`.
> ```

same here



_Code quote:_
```Markdown
.
```

**Review by xpconanfan — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly/846)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @eric100lin and @mhaoli)

___
*[`CHANGELOG.md` line 26 at r1](https://reviewable.io/reviews/google/mobly/846#-N8REEFw4ySOcbCTX1sl:-N8RgQQ5E3bvN2e4EAZq:b-896fix) ([raw file](https://github.com/google/mobly/blob/e20286b6f912138a85dc8b0a3f20724b8b12d628/CHANGELOG.md#L26)):*
<details><summary><i>Previously, mhaoli (Minghao Li) wrote…</i></summary><blockquote>

Delete this period, or use periods for all the four list items?
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 30 at r1](https://reviewable.io/reviews/google/mobly/846#-N8RERY1E0FR-qR8L30c:-N8RgR0t437hrh3TDLQe:b-896fix) ([raw file](https://github.com/google/mobly/blob/e20286b6f912138a85dc8b0a3f20724b8b12d628/CHANGELOG.md#L30)):*
<details><summary><i>Previously, mhaoli (Minghao Li) wrote…</i></summary><blockquote>

same here
</blockquote></details>

Done.

**Review by mhaoli — APPROVED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/mobly/846)* status: all files reviewed, 2 unresolved discussions (waiting on @eric100lin and @mhaoli)

**Review by mhaoli — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly/846)* status: 2 of 4 files reviewed, all discussions resolved (waiting on @eric100lin)

File: mobly/base_test.py [hunk 1]
```diff
@@ -838,21 +838,22 @@ def exec_one_test(self, test_name, test_method, record=None):
...
-  def _assert_function_name_in_stack(self, expected_func_name):
-    """Asserts that the current stack contains the given function name."""
+  def _assert_function_names_in_stack(self, expected_func_names):
+    """Asserts that the current stack contains any of the given function names.
+    """
     current_frame = inspect.currentframe()
     caller_frames = inspect.getouterframes(current_frame, 2)
     for caller_frame in caller_frames[2:]:
-      if caller_frame[3] == expected_func_name:
+      if caller_frame[3] in expected_func_names:
```
Comment (mhaoli) [inline mobly/base_test.py:847]: Sorry, why is this file modified again? I think the modification is merged into master.

This PR should only modify 2 files, but I see this PR modified 4 files, which is strange. Is this because I'm viewing the diff in the wrong way?

**Review by mhaoli — CHANGES_REQUESTED** [review]

File: mobly/base_test.py (hunk 1 above)
Comment (mhaoli) [reply mobly/base_test.py:847]: Resolving.

This is just because I'm viewing the diff in the wrong way.

**Review by mhaoli — APPROVED** [review]
